src/test/java/com/redhat/cloud/custompolicies/app/RestApiTest.java

```diff
@@ -195,7 +195,23 @@ void storeNewPolicy() {
     assert headers.hasHeaderWithName("Location");
     // TODO extract id and then check in subsequent call
     // that the policy is stored
```

Comment:
I think you already did this on this PR :)

Reply:
Was only checking if someone pays attention :-)